webpack/assets/javascripts/react_app/components/notifications/Notification.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Icon from '../common/Icon';
+import moment from 'moment';
+
+/* eslint-disable camelcase */
+const Notification = ({created_at, seen, text, level}) => {
+  const created = moment(created_at);
+  const markup = seen ?
+    (<span className="drawer-pf-notification-message">{text}</span>) :
+    (<strong className="drawer-pf-notification-message">{text}</strong>);
+
+  return (
+    <div className="drawer-pf-notification">
+      <Icon type={level} css="pull-left"></Icon>
+      {markup}
+      <div className="drawer-pf-notification-info">
+        <span className="date">{created.format('M/D/YY')}</span>
```

Comment:
The mark as read still needs to be implemented. Regarding the action, we need to flesh out how that would work. PF demo and code only show pseudo actions